### ISSUE TITLE
Alpha 1 Moderation

### ISSUE THREAD

**Issue by wpillar** [mybb/mybb2#45]
**Features**
- [x] Soft delete post
- [x] Hard delete post
- [x] Soft delete topic
- [x] Hard delete topic
- [x] Close / open topic
- [x] Approve / unapprove posts
- [x] Approve / unapprove topics
- [x] Move posts to existing topic
- [x] Move topics to other forums
- [x] Merge posts of the same author
- [x] Stick / unstick topics
- [x] Moderation Control Panel
  - [x] Moderation queue
  - [x] Moderation logs

Thoughts on the above and things I've missed?

**Comment by premiumcuyuz** [comment]
all features fixed?

**Comment by euantorano** [comment]
@premiumcuyuz Not quite, the JavaScript is still broken, and I don't believe topic level moderation is implemented yet.

We've also been discussing renaming things and having an approval queue in the moderator control panel (which isn't yet created).
